src/commands/upload.js

```diff
@@ -3,13 +3,16 @@ const utils = require('../utils');
 
 const upload = context => {
   context.line('Preparing to upload a new version.\n');
-  return utils.upload().then(() => {
-    context.line(
-      `\nUpload of ${constants.BUILD_PATH} and ${
-        constants.SOURCE_PATH
-      } complete! Try \`zapier versions\` now!`
-    );
-  });
+  return utils
```

Comment:
while you're in here, do you mind converting this to `async`? we've been doing that with other commands as we go, which will eventually remove our dependence on babel.

Reply:
I can definitely do that! Curious though—what's preventing us from dropping babel now? If it's just waiting to convert to `async/await`, I would think we could run the CLI in node natively.